src/index.js

```diff
@@ -2,20 +2,10 @@ import React, { Component } from "react";
 import ReactDom from "react-dom";
 import TestUtils from "react-dom/test-utils";
 
-class StatelessWrapper extends Component {
-  render() {
-    return this.props.children; // eslint-disable-line react/prop-types
-  }
-}
-
-export function mount(component) {
-  const rendered =
-    TestUtils.renderIntoDocument(component) ||
-    TestUtils.renderIntoDocument(
-      <StatelessWrapper>{component}</StatelessWrapper>
-    );
-
-  return ReactDom.findDOMNode(rendered);
+export function mount(element) {
+  const div = document.createElement('div');
+  ReactDOM.render(element, div);
```

Comment:
`ReactDOM` -> `ReactDom`, or rename the import.

Reply:
Let me know what you think about my other comment and I can update this.

Reply:
Fixed